js/client/modules/@arangodb/arango-database.js

```diff
@@ -986,6 +986,20 @@ ArangoDatabase.prototype._createStatement = function (data) {
   return new ArangoStatement(this, data);
 };
 
+
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief factory method to create and execute a new statement
+// //////////////////////////////////////////////////////////////////////////////
+
+ArangoDatabase.prototype._executeJson = function (plan, options = {}) {
+  let command = `
+        let data = ${JSON.stringify(plan)};
+        let opts = ${JSON.stringify(options)}
+        return AQL_EXECUTEJSON(data);
+      `;
+  return this._connection.POST("/_admin/execute", command).json;
+};
```

Comment:
This function should not be offered as part of our public API in arangosh. It is a function that is only used from tests and was never advertised.

Reply:
we will eventually have to offer this API - at least for test purposes.

Reply:
We can move it into a helper file. But it should not appear as a suggestion when using auto-completion on the `db` object.

Reply:
Why not put this into `aql-helper.js` as some of the other test-only helper functions?